themes/bootstrap3/templates/myresearch/menu.phtml

```diff
@@ -1,3 +1,7 @@
+<?
+  $user = $this->auth()->isLoggedIn();
+  $capabilityParams = $user ? $user->toArray() : [];
```

Comment:
What is supposed to be going on here? The second parameter of checkCapability is supposed to be an array containing the parameters that would be passed to the function listed in the first parameter... so, for example, if you would call a function as `function($param1, $param2)`, you would check it with `checkCapability('function', [$param1, $param2])`. Here, the user object appears to be getting converted into an array and passed as the argument list. I don't think that's what we want to be doing, but I'm not sure what the intent is. Most ILS drivers ignore the second parameter anyway, so I want to be sure we don't end up doing something inconsistent that seems to work now but causes problems and confusion down the road.

Reply:
Uhh.. well, my bad. Should be fixed now. The ILS patron was the one that should be passed to the checkCapability call.